### ISSUE TITLE
RawRecord returns partial row.

### ISSUE THREAD

**Issue by CodeCheetah** [JoshClose/CsvHelper#1061]
When using ReadingContext.RawRecord in BadDataFound it returns a partial row data where error occurs and does not return an entire unprocessed line.

For example, a line in the CSV file:
`"tmmx"><script>alert(1)</script>","tmmx",""><script>alert(1)</script>","",""><script>alert(1)</script>","asd"><script>alert(1)</script>","",""><script>alert(1)</scrip"`

RawRecord only returns:
`"tmmx"><script>alert(1)</script>","tmmx",""`

**Comment by JoshClose** [comment]
It stops where the error was found. If there is an error, it can't reliably finish the row.

**Comment by CodeCheetah** [comment]
Is there a property that returns an entire row (raw line) even if there is an error for instance to manually parse with regex, write to a log or something?

**Comment by JoshClose** [comment]
No. If there error is due to unmatched quotes or something, it could return the whole rest of the file. When an error is detected, the parser can't move past it's current position.
